client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/patterns-data.ts

```diff
@@ -202,359 +168,287 @@ const useFooterPatterns = () => {
 };
 
 const useSectionPatterns = () => {
-	const intent = useIntent();
-	const translate = useTranslate();
-
-	const about = translate( 'About' );
-	const callToAction = translate( 'Call to action' );
-	const images = translate( 'Images' );
-	const links = translate( 'Links' );
-	const list = translate( 'List' );
-	const numbers = translate( 'Numbers' );
-	const portfolio = translate( 'Portfolio' );
-	const posts = translate( 'Posts' );
-	const services = translate( 'Services' );
-	const testimonials = translate( 'Testimonials' );
+	const about = 'about';
+	const callToAction = 'call-to-action';
+	const images = 'images';
+	const links = 'links';
+	const list = 'list';
+	const numbers = 'numbers';
+	const portfolio = 'portfolio';
+	const posts = 'posts';
+	const services = 'services';
+	const testimonials = 'testimonials';
 
-	const sectionPatterns: Pattern[] = useMemo( () => {
-		let patterns = [
+	const sectionPatterns: Pattern[] = useMemo(
```

Comment:
We'll replace the list of sections with a request to `dotcompatterns` in a following PR